docs/requirements/requirements.md

```diff
@@ -205,22 +232,83 @@ This document is in the making since we are still in the requirements engineerin
 - There should be a kind of demo mode so that a new user can try out the system quickly and easily: We provide test data with which the system can be directly *loaded*.
 - Origin: Developers
 
+#### R-U-015: Repeatability of experiments
+
+- The system must ensure that researchers can repeat experiments in a controlled environment as often as they wish.
+- As a researcher, I want to repeat an experminent multiple times and I need the environment to be the exact same for each repetition of the experiment.
+- A stakeholder said: "If I have scaling in place, when the experiment is finished, you want the state to be just like before the experiment. This allows repeatability."
+- This requirement could also be moved into the section Configurations.
+- Origin: Researcher *Toni*
+
+#### R-U-016: Scheduling of experiments
+
+- The system must offer the possibility to schedule experiments.
+- As a researcher, I want to be able to schedule experiments.
+- This requirement could also be moved into the section Configurations.
+- Origin: Researcher *Toni*
+
 ### Non-functional Requirements
 
 #### R-NFR-001: Performance
 
 - The microservices must be performant.
 - Origin: Examiner
 
-#### R-NFR-002: Elasticity
+#### R-NFR-002: Elasticity / Auto scaling
 
 - The system must be *elastic*.
-- Origin: Examiner
+- The system must scale automatically.
+- Origin: Examiner, Researcher *Toni*
 
 ##### Open Question
 
 - What exactly does *elastic* mean? What is the examiner's understanding of the term?
 
+#### R-NFR-003: Resilience
+
+- The system should be resilient.
```

Comment:
Is there a way we can be more specific about what circumstances the system should be able to withstand?

Reply:
I think we have to conduct our stakeholders once more in order to clarify this. Right now, we are lacking detailed information to make more specific statement. The stakeholder said that the services should not disappear all of a sudden -- I can add that to the requirement.